roles/alertmanager/tasks/main.yml

```diff
@@ -47,3 +47,15 @@
     - alertmanager
     - configure
     - alertmanager_configure
+
+- name: Ensure alertmanager is enabled on boot
+  become: true
+  ansible.builtin.systemd:
+    daemon_reload: true
+    name: alertmanager
+    enabled: true
+    state: started
+  when:
+    - not ansible_check_mode
```

Comment:
The systemd module has full support for check_mode so I don't think this is needed
```suggestion
```

Reply:
Thanks for the review. I've modified and pushed the part you suggested.